SPECS/prebuilt-ca-certificates-base/prebuilt-ca-certificates-base.spec

```diff
@@ -2,8 +2,8 @@ Summary:        Prebuilt version of ca-certificates-base package.
 Name:           prebuilt-ca-certificates-base
 # When updating, "Epoch, "Version", AND "Release" tags must be updated in the "ca-certificates" package as well.
 Epoch:          1
```

Comment:
The comment above says that Epoch also to be updated whenever Version and Release fields are updated.